Makefile

```diff
@@ -94,13 +94,29 @@ deps:
 	cargo install --version 1.14.0 hyperfine
 	cargo install --version 0.9.49 cargo-nextest
 	cargo install --version 0.5.9 cargo-llvm-cov
-	pyenv install pypy3.7-7.3.9
-	pyenv global pypy3.7-7.3.9
+	cargo install --version 0.11.0 wasm-pack
+	pyenv install pypy3.9-7.3.9
+	pyenv local pypy3.9-7.3.9
 	pip install -r requirements.txt
-	pyenv install 3.7.12
-	pyenv global 3.7.12
+	pyenv install 3.9.15
+	pyenv local 3.9.15
```

Comment:
It's better to use the `PYENV_VERSION` env var instead of changing local each time.

Reply:
I added the cairo-rs-env and cairo-rs-pypy-env
[0b18463](https://github.com/lambdaclass/cairo-rs/pull/963/commits/0b1846363793f5a449fc99e7be20f35901051488)